### PR TITLE
Make wait list visible on user info page & minor fix

### PR DESCRIPTION
**Pull request by lil1an**

### CODE REVIEW

File: components/user/CourseCard.tsx [hunk 1]
```diff
@@ -1,8 +1,7 @@
-import useCourse from "@hooks/useCourse"
-import Link from "next/link"
+import useCourse from '@hooks/useCourse'
+import Link from 'next/link'
 
-
-const CourseCard = ({ code }: { code: string }) => {
+const CourseCard = ({ code }: { code?: string }) => {
```
Comment (jinh0) [inline components/user/CourseCard.tsx:4]: Why did you make `code` optional?

File: components/user/CourseCard.tsx [hunk 2]
```diff
@@ -20,13 +19,16 @@ const CourseCard = ({ code }: { code: string }) => {
   const course = data.result
 
   return (
-    <Link href={`/courses/${code}`} className='flex flex-row gap-x-4 '>
+    <Link
+      href={`/courses/${code}`.toLowerCase()}
```
Comment (jinh0) [inline components/user/CourseCard.tsx:23]: can you change this to `/courses/${code.toLowerCase()}`, that localizes what needs to be lowercased

File: components/user/CourseCard.tsx [hunk 3]
```diff
@@ -20,13 +19,16 @@ const CourseCard = ({ code }: { code: string }) => {
...
+      className='flex flex-row gap-x-4 '
+    >
       <div className='w-60 border rounded-lg p-2 px-4 flex-shrink-0 snap-start'>
-        <p className='font-semibold'>{format(course.code)}</p>
-        <p className=''>{course.name}</p>
+        <p className='font-semibold'>{format(course?.code)}</p>
+        <p className=''>{course?.name}</p>
```
Comment (jinh0) [inline components/user/CourseCard.tsx:28]: Can you explain to me why you added the optional chaining?

**Review by jinh0 — CHANGES_REQUESTED** [review]

File: components/user/CourseCard.tsx (hunk 1 above)
Comment (lil1an) [reply components/user/CourseCard.tsx:4]: I was getting run time errors when calling CourseCard and something about something being undefined. So I changed this to optional. Turns out I just needed the course code to be lowercase. Everything works without making this optional; I just forgot to remove it. I will remove it now.

File: components/user/CourseCard.tsx (hunk 2 above)
Comment (lil1an) [reply components/user/CourseCard.tsx:23]: Okay!

File: components/user/CourseCard.tsx (hunk 3 above)
Comment (lil1an) [reply components/user/CourseCard.tsx:28]: I was testing something for the same reasons mentioned above. All works now without it. I will remove it now.

**Review by jinh0 — APPROVED** [review]

**Pull request merged**